doc/models/iaf.rst

```diff
@@ -0,0 +1,7 @@
+Integrate and fire neuron models
```

Comment:
```suggestion
Integrate-and-Fire (IF) neuron models
```

Reply:
I think integrate-and-fire is usually abbreviated iaf ?
Also for consistency, I'd make 'fire' lower case

Reply:
This is the proper way to write and abbreviate it: see [here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3325187/?tool=pmcentrez&report=abstract) and above response.
One can drop the uppercase to "fire", though.
And IF is also used for consistency with other IF models (aEIF, GIF, EIF...)

Reply:
I do agree we should use the commonly accepted terms, so I don't have a problem committing these changes. We just may need to consider how we use If and IAF in nest.

Reply:
you're right... I never payed attention to this issue... IAF definitely looks wrong, but I may have seen IaF used some times... I'm more used to IF though, but we might have to do a proper survey